server/src/core/database-service.ts

```diff
@@ -39,47 +41,43 @@ export class DatabaseService {
     console.log(`Initializing database service with Qdrant at ${QDRANT_URL}`);
   }
   
-  // Initialize Qdrant and create collection if it doesn't exist
+  // Initialize Qdrant and create collections if they don't exist
   public async initialize(): Promise<void> {
     try {
-      // Check if collection exists
+      // Check if collections exist
       const collections = await this.qdrantClient.getCollections();
-      const collectionExists = collections.collections?.some(
+      
+      // Check if the chunks collection exists
```

Comment:
chunking document is behind, under the hood mechanism/technical then we don't need to expose it as naming like this